src/entity/Game.ts

```diff
@@ -0,0 +1,32 @@
+import { Status } from '../types/game'
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  OneToMany,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm'
```

Comment:
Nitpick, but typically you want imports from npm packages to come first in the import list